package.json

```diff
@@ -9,6 +9,7 @@
   "scripts": {
     "build": "env-cmd -e ci,b2c,$SPA_ENV ng build storefrontapp --prod",
     "build:assets": "yarn --cwd ./projects/assets build",
+    "build:cart": "ng build cart --prod",
```

Comment:
Why we are not building schematics with the lib?
Also, we probably should include it in build:libs command (the same with 'test:libs).